resources/user-agents/browsers/chrome/chrome-ios-48-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Chrome #MAJORVER#.#MINORVER# for iOS",
-  "versions": [63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51, 50, 49, 48],
+  "versions": [64, 63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51, 50, 49, 48],
```

Comment:
Why did you not add iOS 11.1 here too?

Reply:
I have it coming in the PR I have prepped for #1666

Reply:
ok